apps/studio/src/store/modules/data/DataModuleBase.ts

```diff
@@ -103,14 +104,67 @@ const buildBasicMutations = <T extends HasId>(sortBy?: SortSpec) => ({
   },
 })
 
-export function mutationsFor<T extends HasId>(obj: any, sortBy?: SortSpec) {
+export function mutationsFor<T extends HasId>(obj: any = {}, sortBy?: SortSpec) {
   return {
     ...buildBasicMutations<T>(sortBy),
     ...obj
   }
 }
 
-export function localActionsFor<T extends ApplicationEntity>(cls: any, other: any, loadOptions: any = {}) {
+export function utilActionsFor<T extends Transport>(type: string, other: any = {}, loadOptions: any = {}) {
+  return {
+    async load(context) {
+      context.commit("error", null);
+      await safely(context, async () => {
+        const items = await Vue.prototype.$util.send(`appdb/${type}/find`, { options: loadOptions });
+        if (context.rootState.workspaceId === LocalWorkspace.id) {
+          context.commit('upsert', items);
+        }
+      })
+    },
+    async poll() {
```

Comment:
SInce this doesn't actually do anything, do we need it?

Reply:
It does stuff in ultimate, this is just to make merging stuff easy I believe